src/library/styles/getResponsiveStyles.js

```diff
@@ -40,18 +40,18 @@ const getMediaQueries = (
   let queries = [];
 
   const getQueryWidth = (value) =>
-    typeof value === 'number' ? value : theme[`breakpoint_${value}`];
+    typeof value === 'number'
+      ? `${value}px`
+      : theme[`breakpoint_${value}`] || value;
```

Comment:
can you put parens around one of these expressions? I forget the ordinality of ternary vs `||` so I'm not sure looking at it, which ought to win

```
( typeof value === 'number'
      ? `${value}px`
      : theme[`breakpoint_${value}`] ) || value;
```
or 
```
typeof value === 'number'
      ? `${value}px`
      : ( theme[`breakpoint_${value}`] || value);
```

Reply:
I added them in, but prettier stripped them right back out.

The latter is correct, for the record.